src/Components/LoginForm.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import {useNavigate} from 'react-router-dom';
+function LoginForm() {
+  const initialValues = { username: "", email: "", password: "" };
+  const [formValues, setFormValues] = useState(initialValues);
+  let navigate=useNavigate();
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormValues({ ...formValues, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    validate(formValues);
+    if(formValues.username!==''&& formValues.email!==''&& formValues.password!=='' && formValues.password.length>4 && formValues.password.length<10)
```

Comment:
Don't need add check here. Just use a boolean and set its state in validate method.